pkg/liqonet/tunnel/metrics/metrics.go

```diff
@@ -48,9 +50,16 @@ func init() {
 		nil,
 	)
 
-	PeerLastHandshake = prometheus.NewDesc(
-		"liqo_peer_last_handshake_seconds",
-		"UNIX timestamp for the last handshake with a given peer.",
+	PeerLatency = prometheus.NewDesc(
+		"liqo_peer_latency_seconds",
```

Comment:
```suggestion
		"liqo_peer_latency_ms",
```

Nit. Milliseconds is probably a bit better.